packages/crypto/src/crypto.ts

```diff
@@ -1,18 +1,103 @@
-import {
-    SNARK_FIELD_SIZE,
-    SnarkBigInt,
-    genRandomSalt,
-    hash5,
-    hashOne,
-    hashLeftRight,
-    stringifyBigInts,
-    unstringifyBigInts,
-} from 'maci-crypto'
+import assert from 'assert'
+import crypto from 'crypto'
+const { stringifyBigInts, unstringifyBigInts } = require('ffjavascript').utils
+
+import { poseidon } from './poseidon'
+
+// Copy from maci-crypto@0.9.1
```

Comment:
Why are we copying the `maci-crypto` code here? I think re-exporting it to simplify use is okay, but copying it means we don't get any upstream patches or fixes.

Reply:
Because it will download all dependencies we don't need it while installation. 😅
I want to decrease the dependencies and we can only maintain what we need